components/list/list.js

```diff
@@ -183,8 +183,7 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 	}
 
 	_getItemByIndex(index) {
-		const items = this.getItems();
-		if (index > items.length - 1) return;
+		const items = this.getItems() || [];
```

Comment:
Are we worried that the behaviour here changed a bit? Like previously if you asked for an item index beyond the bounds it would give you `undefined`. Not it'll throw an exception. It looks like the only place calling this is handling it though, so probably ok.

Reply:
Oh interesting, TIL!